spark-operator-api/src/main/java/org/apache/spark/k8s/operator/spec/ClusterInstanceConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.spark.k8s.operator.spec;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.fabric8.generator.annotation.Required;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ClusterInstanceConfig {
```

Comment:
I didn't compile the code, but `ClusterInstanceConfig` will generate a pattern, `clusterInstanceConfig` like the following. Does this match with your change on `examples/*.yaml`?
```
  clusterTolerations:
    clusterInstanceConfig:
      initWorkers: 1
      minWorkers: 1
      maxWorkers: 1
```